src/main/java/seedu/address/logic/commands/OpenCommissionCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.commission.Commission;
+
+/**
+ * Selects a commission and shows their details.
+ */
+public class OpenCommissionCommand extends Command {
+    public static final String COMMAND_WORD = "opencom";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Opens a commission and shows commission details"
```

Comment:
The phrasing here is a little weird, perhaps we should try to make this clearer?